kupyna/Cargo.toml

```diff
@@ -0,0 +1,30 @@
+[package]
+name = "kupyna"
+version = "0.1.0"
+description = "Hashing algorithm for Kupyna, 8 to 512 length"
+authors = [
+    "Joshua Koudys <josh@qaribou.com>",
+    "Raj Singh <typhoeusxoxo@gmail.com>",
+    "RustCrypto Developers",
```

Comment:
We generally use the "RustCrypto Developers" umbrella authorship. You can explicitly indicate particular authors in the LICENSE-MIT file.